src/main/java/seedu/address/model/delivery/Note.java

```diff
@@ -9,7 +9,7 @@
 public class Note {
 
     public static final String MESSAGE_CONSTRAINTS =
-        "Note should not be a non-empty alphanumeric string";
+        "Note should be a non-empty alphanumeric string";
```

Comment:
Since this error message is for the user, should we avoid in using "string" and align with our User Guide in saying "The note must consist of alphanumeric characters and cannot be empty."?